packages/backend/src/views/story.njk

```diff
@@ -197,7 +197,12 @@ AFRAME.registerComponent('exit', {
    </a-gui-flex-container>
 
    <a-camera position="0 0 6">
-      <a-cursor />
+      <a-entity position="0 0 -3"
+                geometry="primitive: ring; radiusInner: 0.1; radiusOuter: 0.15;"
+                material="color: #FF69B4; shader: flat"
```

Comment:
#FF69B4 is `hotpink`.

* [HTML colours](https://htmlcolorcodes.com/color-names/)